src/app/Table.elm

```diff
@@ -1,29 +1,91 @@
-module Table exposing (view, init, Msg)
+module Table exposing (view, init, initWithPlayersAndCup)
+
+{-|
+
+   This module handles the table of players,
+   and everything relating to that.
+
+-}
 
 import Html exposing (..)
+import Html.Events exposing (..)
+import Types exposing (PlayerID)
+import TableMsg exposing (Msg(..))
+import Dice
 
 
-{-
+type alias PlayerName =
+    String
 
-   This module handles the view rendering for the table of players
-   and everything relating to that.
 
--}
+type alias DiceCount =
+    Int
+
 
+type alias TablePlayer =
+    { id : PlayerID
+    , name : PlayerName
+    , diceCount : DiceCount
+    }
 
-type Model
-    = Model
 
+type alias TablePlayers =
+    List TablePlayer
 
-type Msg
-    = Msg
+
+type alias Model =
+    { tablePlayers : TablePlayers
+    , myCup : Dice.Cup
+    }
 
 
 init : Model
 init =
-    Model
+    { tablePlayers = noPlayers
+    , myCup = Dice.emptyCup
+    }
+
+
+initWithPlayersAndCup : List ( Int, String, Int ) -> List Int -> Model
+initWithPlayersAndCup tuples cup =
+    let
+        tupleToPlayer ( id, name, diceCount ) =
+            { id = id, name = name, diceCount = diceCount }
+    in
+        { tablePlayers = List.map tupleToPlayer tuples
+        , myCup = Dice.cupFromIntList cup
+        }
+
+
+noPlayers : TablePlayers
+noPlayers =
+    []
+
+
+testPlayers : TablePlayers
+testPlayers =
+    [ TablePlayer 1 "Joe" 6, TablePlayer 2 "Garrett" 5 ]
+
+
+view : (Msg -> msg) -> Model -> Html msg
```

Comment:
Looks like you've found the [translator pattern](https://medium.com/@alex.lew/the-translator-pattern-a-model-for-child-to-parent-communication-in-elm-f4bfaa1d3f98#.i3omecs1p):
`type alias Translator parentMsg = Msg -> parentMsg`

I'd strongly suggest our solution reflects the problem we're trying to model. This feels like over-engineering for one page with 2-3 buttons.
There is also the consideration of keeping it simple for Kartik or beginners in general